### PR TITLE
Update dep takebuf_array method

### PR DESCRIPTION
**Pull request by musm**
Please tag after a merge, thanks

### CODE REVIEW

File: src/ssl.jl [hunk 1]
```diff
@@ -224,7 +224,7 @@ function Base.readavailable(ctx::SSLContext)
     b = IOBuffer()
     write(b, read(ctx, 1))
     write(b, read(ctx, nb_available(ctx)))
-    return takebuf_array(b)
+    return take!(b)
```
Comment (malmaud) [inline src/ssl.jl]: Shouldn't it be `@compat take!(b)`?

Comment (musm) [inline src/ssl.jl]: no, see https://github.com/JuliaLang/Compat.jl/blob/master/src/Compat.jl#L1159

**Comment by malmaud** [comment]
Ah, OK.

File: src/ssl.jl (hunk 1 above)
Comment (tkelman) [inline src/ssl.jl]: you do need to use or import Compat though

**Comment by musm** [comment]
g2g

**Comment by musm** [comment]
Is this ok? Thx

**Pull request merged**